components/datetime/src/skeleton/helpers.rs

```diff
@@ -81,6 +81,7 @@ const REQUESTED_SYMBOL_MISSING: u32 = 10000;
 /// there is no attempt to add on missing fields. This enum encodes the variants for the current
 /// search for a best skeleton.
 #[derive(Debug, PartialEq, Clone)]
+#[allow(clippy::exhaustive_enums)] // this type is stable
```

Comment:
Thought: I don't know why this is exported. It looks like something that should be internal.

Reply:
It's doc(hidden), i stuck the allow() at the top level instead